docs/content/content/content-principles.mdx

```diff
@@ -7,7 +7,7 @@ order: 1
 
 You can use these standards to guide you to create content that is clear and effective.
 
-Good content reduces the cognitive load on users and helps them achieve their aims more easily.
+Good content reduces the cognitive load on users. It helps them achieve their aims more easily.
```

Comment:
Weirdly, having a new sentence for "It" seems to increase cognitive load for me 😅

Reply:
@Suzanne-aff As discussed